### ISSUE TITLE
mobileconfig file for Bluetooth and accesibilty reports "Must be a well formed XML string"

### ISSUE THREAD

**Issue by ReySys** [microsoft/mdatp-xplat#153]
After Uploading reports "Must be a well formed XML string"

![Image](https://github.com/user-attachments/assets/1d85e0d1-15bf-4c3f-83d1-aae34fba0a12)

**Comment by maximvelichko** [comment]
@ReySys, this is correct, it must be a well-formed XML string, you are attempting to upload an HTML.
Follow documentation for your MDM solution (Intune) on how to import configuration profiles.
